src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.props

```diff
@@ -42,6 +43,7 @@
     <OutputType Condition=" '$(OutputType)' == 'Exe' ">Library</OutputType>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(AndroidApplication)' == 'true' ">
+    <ProduceReferenceAssembly Condition=" '$(ProduceReferenceAssembly)' == '' ">false</ProduceReferenceAssembly>
```

Comment:
@rolfbjarne I don't know if you are seeing lots of extra files for iOS projects, but you might consider `GenerateDependencyFile=false` by default and `ProduceReferenceAssembly=false` by default for iOS application projects.

Reply:
Good idea, I had `GenerateRuntimeConfigurationFiles=false` set somewhere too at some point, which iirc gets rid of *.runtimeconfig.json